packages/0xcert-conventions/src/assets/86-base-asset.ts

```diff
@@ -15,14 +15,14 @@ export interface Object86 extends ObjectErc721 {
  * Base asset data object JSON Schema.
  */
 export const schema86 = {
-  $schema: 'http://json-schema.org/draft-07/schema',
```

Comment:
This is a json schema the $ here has a different meaning. There should be no change. Please fix this in other json schema instances and tests as well.